calc/src/App.css

```diff
@@ -0,0 +1,133 @@
+body {
+  background-color: #e5e5e5;
+}
+button {
+  border-radius: 0 0 0 0;
+  width: 72px;
+  height: 62px;
+  background-color: #384351;
+  color: #d1d6db;
+  border: transparent;
+  font-size: 19px;
+}
+
+#App {
+  position: absolute;
+  width: 300px;
+  height: 514px;
+  background-color: pink;
+  left: 50%;
+  top: 50%;
+  transform: translate(-50%, -50%);
+}
+form {
+  margin: 0px;
+}
+#calc {
+  height: 498px;
+  margin-top: 6px;
+  margin-bottom: 10px;
+  margin-right: 8px;
+  margin-left: 6px;
+}
+#form {
+  width: 288px;
+  height: 188px;
+  border-radius: 15px 15px 0px 0px;
+  background-color: #46515f;
+  border: 0px;
+}
+#input {
+  font-size: 25px;
+  text-align: right;
+  color: #f1f2f3;
+}
+
+#buttons {
+  border-bottom-right-radius: 15px;
+  border-bottom-left-radius: 15px;
+  background-color: #384351;
+  width: 288px;
+  height: 310px;
+}
+#del,
+#seven,
+#four,
+#one {
+  width: 72px;
+  height: 62px;
+  cursor: pointer;
+}
+#zero {
+  width: 72px;
+  height: 62px;
+  border-bottom-left-radius: 15px;
+}
+#pm {
+  width: 72px;
+  height: 62px;
+
+  margin-left: 72px;
+  margin-top: -310px;
+}
+#eight,
+#five,
+#two {
+  width: 72px;
+  height: 62px;
+
+  margin-left: 72px;
+}
+#perc {
+  width: 72px;
+  height: 62px;
+
+  margin-left: 144px;
+  margin-top: -248px;
+}
+#nine,
+#six,
+#three,
+#comma {
+  width: 72px;
+  height: 62px;
+
+  margin-left: 144px;
+}
+#dzie {
+  width: 72px;
+  height: 62px;
+  margin-left: 216px;
+  margin-top: -310px;
+}
+#multi,
+#sub,
+#add,
+#equal {
```

Comment:
Maybe instead of creating ID for every the same component we should add 1 single class and reuse it?